### ISSUE TITLE
[DX] installation has 4 commands, which is a lot

### ISSUE THREAD

**Issue by tabacitu** [Laravel-Backpack/CRUD#5081]
# Bug report

### What I did

Followed [the v6 docs to install Backpack](https://backpackforlaravel.com/docs/6.x/installation).

### What I expected to happen

Be easy - one-two commands max.

### What happened

It's a whopping four commands:

```
composer require backpack/crud:"v6.x-dev"
php artisan storage:link
php artisan backpack:install
php artisan basset:internalize
```

### What I've already tried to fix it

Nothing yet. But I think we should:
- 🔴 MUST automatically do `php artisan storage:link` upon installation; 
- 🟡 but then again... `php artisan storage:link` is something the developer should do _upon each installation_... so how about we run it (sure) but also prompt the developer, ask if they want us to add it to `composer.json` scripts, under `post-install-cmd`? that way we make sure the dev runs `php artisan storage:link`, at least upon first deployment; fewer headaches on deployment;
- 🔴 MUST run `php artisan basset:internalize` as a last step in the Backpack/CRUD installation process; and maybe in all our `backpack:require-x` commands too? 🤷‍♂️

**Comment by tabacitu** [comment]
Note:
- I’ve followed the docs to install a clean v6, which instruct me to do “php artisan basset:internalize” first, then “php artisan backpack:install”. That’s done in order for the last command to be actionable, to show the final URL people should click and get started with their admin panel. But there’s a problem… because theme-tabler is only installed on the install command, that means the theme assets never got internalized. No bueno. Perhaps we should run “php artisan basset:internalize” as part of the v6 installation command… right before we show all the final output?

**Comment by tabacitu** [comment]
Hmm... now that we're running `basset:install` and BASSET_DEV_MODE is true by default... do we really need to run `php artisan basset:internalize` on local?! I guess we should still do that, for zip files etc? 🤷‍♂️ What do you think @promatik ? 

I guess what we want is a pleasant first experience loading Basset pages. Are most pages fast right now, because we're using BASSET_DEV_MODE? If so... perhaps it's a better experience to NOT internalize because that takes quite a bit of time and it makes the installation process longer.

**Comment by pxpm** [comment]
Also, can we remove zips at least from core ? 

We can use 5 or 6, 100 @basset() directives to load the needed files, but this tinymce zip is killing me.

**Comment by tabacitu** [comment]
> Also, can we remove zips at least from core ?
> We can use 5 or 6, 100 https://github.com/basset() directives to load the needed files, but this tinymce zip is killing me.

I don't think this is regarding the installation - please open a separate issue for this. 

---

I'm reasonably happy with how things are right now. We've gone from:
```
composer require backpack/crud:"v6.x-dev"
php artisan storage:link
php artisan backpack:install
php artisan basset:internalize
```
to
```
composer require backpack/crud:"v6.x-dev"
php artisan backpack:install
```

Which I think is the best we can do, really. So I'm going to close this issue.
